src/app/utils/getNews.ts

```diff
@@ -0,0 +1,31 @@
+import { schemaNewsResponse } from '@/models/schemes';
+import { NewsResponse, NewsWithId } from '@/models/types';
+import axios from 'axios';
+import { z } from 'zod';
+
+const getNews = async (keyword: string) => {
+  try {
+    const response = await axios.get<NewsResponse>(
```

Comment:
You don’t need to set `NewsResponse` type here, because you have `zod` which will tell the `ts` what type it should be